neqo-transport/src/connection.rs

```diff
@@ -1218,7 +1266,13 @@ impl Connection {
                 path.local_cid(),
             );
 
-            PacketBuilder::long(encoder, pt, path.remote_cid(), path.local_cid())
+            PacketBuilder::long(
+                encoder,
+                pt,
+                path.remote_cid(),
+                path.local_cid(),
+                quic_version,
```

Comment:
Ordering might be important here.  Perhaps we should have this match the order of fields, so: pt, version, dcid, scid

Reply:
So I've been putting quic_version as a param last because I couldn't come up with a consistent param ordering scheme, and had a thought that maybe someday we could remove this and it would be easier to remove if it were always last. But the more I think about it, that increasingly seems like a pipe dream. So ok I'll reorder.

Reply:
done